diff --git a/node_modules/@nuxtjs/google-adsense/dist/module.mjs b/node_modules/@nuxtjs/google-adsense/dist/module.mjs
index 1993df5..ff819c3 100644
--- a/node_modules/@nuxtjs/google-adsense/dist/module.mjs
+++ b/node_modules/@nuxtjs/google-adsense/dist/module.mjs
@@ -15,7 +15,7 @@ function initializeAdClient(options) {
   return createScriptMeta(
     `adsbygoogle.onload = function() {
       adsbygoogle.pauseAdRequests=${options.pauseOnLoad ? "1" : "0"};
-      [].forEach.call(document.getElementsByClassName('adsbygoogle'), function() { adsbygoogle.push(${adsenseScript}); })
+       [].forEach.call(document.querySelectorAll("ins.adsbygoogle[data-ad-client='${options.id}']"), function() { adsbygoogle.push(${adsenseScript}); })
     };`
   );
 }
