diff --git a/node_modules/@nuxt/image/dist/runtime/components/nuxt-img.mjs b/node_modules/@nuxt/image/dist/runtime/components/nuxt-img.mjs
index 61b828a..1a95631 100644
--- a/node_modules/@nuxt/image/dist/runtime/components/nuxt-img.mjs
+++ b/node_modules/@nuxt/image/dist/runtime/components/nuxt-img.mjs
@@ -66,6 +66,7 @@ export default defineComponent({
       useHead({
         link: [{
           rel: "preload",
+          fetchPriority: "high",
           as: "image",
           nonce: props.nonce,
           ...!isResponsive ? { href: src.value } : {
diff --git a/node_modules/@nuxt/image/dist/runtime/components/nuxt-picture.mjs b/node_modules/@nuxt/image/dist/runtime/components/nuxt-picture.mjs
index d11e82e..6b4a814 100644
--- a/node_modules/@nuxt/image/dist/runtime/components/nuxt-picture.mjs
+++ b/node_modules/@nuxt/image/dist/runtime/components/nuxt-picture.mjs
@@ -49,6 +49,7 @@ export default defineComponent({
     if (props.preload) {
       const link = {
         rel: "preload",
+        fetchPriority: "high",
         as: "image",
         imagesrcset: sources.value[0].srcset,
         nonce: props.nonce,
diff --git a/node_modules/@nuxt/image/dist/runtime/providers/weserv.mjs b/node_modules/@nuxt/image/dist/runtime/providers/weserv.mjs
index d5123f1..2a80517 100644
--- a/node_modules/@nuxt/image/dist/runtime/providers/weserv.mjs
+++ b/node_modules/@nuxt/image/dist/runtime/providers/weserv.mjs
@@ -19,7 +19,7 @@ const operationsGenerator = createOperationsGenerator({
     contrast: "con",
     blur: "blur",
     mirror: "flop",
-    rotate: "rot",
+    rotate: "ro",
     mask: "mask",
     maskTrim: "mtrim",
     maskBackground: "mbg"
