diff --git a/node_modules/@nuxtjs/google-adsense/dist/module.mjs b/node_modules/@nuxtjs/google-adsense/dist/module.mjs
index 1993df5..5385a62 100644
--- a/node_modules/@nuxtjs/google-adsense/dist/module.mjs
+++ b/node_modules/@nuxtjs/google-adsense/dist/module.mjs
@@ -15,7 +15,7 @@ function initializeAdClient(options) {
   return createScriptMeta(
     `adsbygoogle.onload = function() {
       adsbygoogle.pauseAdRequests=${options.pauseOnLoad ? "1" : "0"};
-      [].forEach.call(document.getElementsByClassName('adsbygoogle'), function() { adsbygoogle.push(${adsenseScript}); })
+       [].forEach.call(document.querySelectorAll("ins.adsbygoogle[data-ad-client='${options.id}']"), function() { adsbygoogle.push(${adsenseScript}); })
     };`
   );
 }
@@ -28,7 +28,7 @@ function createScriptMeta(script) {
   };
 }
 
-const ADSENSE_URL = "//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js";
+const ADSENSE_URL = "https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js";
 const TEST_ID = "ca-google";
 const module = defineNuxtModule({
   meta: {
