diff --git a/node_modules/@nuxt/image/dist/runtime/components/nuxt-img.mjs b/node_modules/@nuxt/image/dist/runtime/components/nuxt-img.mjs
index 3ac97b9..3a8bb19 100644
--- a/node_modules/@nuxt/image/dist/runtime/components/nuxt-img.mjs
+++ b/node_modules/@nuxt/image/dist/runtime/components/nuxt-img.mjs
@@ -65,6 +65,7 @@ export default defineComponent({
         link: [{
           rel: "preload",
           as: "image",
+          fetchPriority: "high",
           nonce: props.nonce,
           ...!isResponsive ? { href: src.value } : {
             href: sizes.value.src,
diff --git a/node_modules/@nuxt/image/dist/runtime/components/nuxt-picture.mjs b/node_modules/@nuxt/image/dist/runtime/components/nuxt-picture.mjs
index c471a5e..5379ed8 100644
--- a/node_modules/@nuxt/image/dist/runtime/components/nuxt-picture.mjs
+++ b/node_modules/@nuxt/image/dist/runtime/components/nuxt-picture.mjs
@@ -49,6 +49,7 @@ export default defineComponent({
     if (props.preload) {
       const link = {
         rel: "preload",
+        fetchPriority: "high",
         as: "image",
         imagesrcset: sources.value[0].srcset,
         nonce: props.nonce
