diff --git a/node_modules/@storipress/vue-advertising/dist/components/AdvertisingProvider.vue b/node_modules/@storipress/vue-advertising/dist/components/AdvertisingProvider.vue
index 95f8b2a..ea132ee 100644
--- a/node_modules/@storipress/vue-advertising/dist/components/AdvertisingProvider.vue
+++ b/node_modules/@storipress/vue-advertising/dist/components/AdvertisingProvider.vue
@@ -30,12 +30,13 @@ const advertising = shallowRef<Advertising | null>(new Advertising(config, plugi
 const activate = computed(() => advertising.value?.activate.bind(advertising.value))
 const nowConfig = ref<TAdvertisingConfig | null>(config)
 
-useHead({
-  script: [
-    { async: true, src: 'https://securepubads.g.doubleclick.net/tag/js/gpt.js' },
-    isPrebid ? { async: true, src: 'https://acdn.adnxs.com/prebid/not-for-prod/1/prebid.js' } : {},
-  ],
-})
+// Already loaded in nuxt.config.js
+// useHead({
+//   script: [
+//     { async: true, src: 'https://securepubads.g.doubleclick.net/tag/js/gpt.js' },
+//     isPrebid ? { async: true, src: '/js/prebid.js' } : {},
+//   ],
+// })
 
 useProvideAdvertisingContext({ activate, config: nowConfig })
 
