diff --git a/node_modules/@uploadcare/upload-client/dist/esm/index.browser.mjs b/node_modules/@uploadcare/upload-client/dist/esm/index.browser.mjs
index b75311a..5e04bd6 100644
--- a/node_modules/@uploadcare/upload-client/dist/esm/index.browser.mjs
+++ b/node_modules/@uploadcare/upload-client/dist/esm/index.browser.mjs
@@ -843,17 +843,20 @@ const uploadDirect = (file, { publicKey, fileName, baseURL, secureSignature, sec
         metadata
     })
         .then(({ file }) => {
-        return isReadyPoll(file, {
-            publicKey,
-            baseURL,
-            source,
-            integration,
-            userAgent,
-            retryThrottledRequestMaxTimes,
-            retryNetworkErrorMaxTimes,
-            onProgress,
-            signal
-        });
+            // We don't need the file info for our use, and this avoids chargeable API calls.
+            return file.detail
+
+        //     return isReadyPoll(file, {
+        //     publicKey,
+        //     baseURL,
+        //     source,
+        //     integration,
+        //     userAgent,
+        //     retryThrottledRequestMaxTimes,
+        //     retryNetworkErrorMaxTimes,
+        //     onProgress,
+        //     signal
+        // });
     })
         .then((fileInfo) => new UploadcareFile(fileInfo, { baseCDN }));
 };
