diff --git a/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js b/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js
index 9afcea6..7eccf32 100644
--- a/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js
+++ b/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js
@@ -36,7 +36,7 @@ const nodeOps = {
   setElementText: (el, text) => {
     el.textContent = text;
   },
-  parentNode: (node) => node.parentNode,
+  parentNode: (node) => node ? node.parentNode : null,
   nextSibling: (node) => node.nextSibling,
   querySelector: (selector) => doc.querySelector(selector),
   setScopeId(el, id) {
