diff --git a/node_modules/@vue/shared/dist/shared.cjs.js b/node_modules/@vue/shared/dist/shared.cjs.js
index 55bf448..43c05fb 100644
--- a/node_modules/@vue/shared/dist/shared.cjs.js
+++ b/node_modules/@vue/shared/dist/shared.cjs.js
@@ -331,7 +331,7 @@ function escapeHtml(string) {
 }
 const commentStripRE = /^-?>|<!--|-->|--!>|<!-$/g;
 function escapeHtmlComment(src) {
-  return src.replace(commentStripRE, "");
+  return src.replace ? src.replace(commentStripRE, "") : '';
 }
 
 function looseCompareArrays(a, b) {
diff --git a/node_modules/@vue/shared/dist/shared.cjs.prod.js b/node_modules/@vue/shared/dist/shared.cjs.prod.js
index 3119720..092340b 100644
--- a/node_modules/@vue/shared/dist/shared.cjs.prod.js
+++ b/node_modules/@vue/shared/dist/shared.cjs.prod.js
@@ -331,7 +331,7 @@ function escapeHtml(string) {
 }
 const commentStripRE = /^-?>|<!--|-->|--!>|<!-$/g;
 function escapeHtmlComment(src) {
-  return src.replace(commentStripRE, "");
+  return src.replace ? src.replace(commentStripRE, "") : '';
 }
 
 function looseCompareArrays(a, b) {
