diff --git a/node_modules/@vue/shared/dist/shared.cjs.js b/node_modules/@vue/shared/dist/shared.cjs.js
index f4d48ee..e073f38 100644
--- a/node_modules/@vue/shared/dist/shared.cjs.js
+++ b/node_modules/@vue/shared/dist/shared.cjs.js
@@ -375,7 +375,7 @@ function escapeHtml(string) {
 // https://www.w3.org/TR/html52/syntax.html#comments
 const commentStripRE = /^-?>|<!--|-->|--!>|<!-$/g;
 function escapeHtmlComment(src) {
-    return src.replace(commentStripRE, '');
+    return src.replace ? src.replace(commentStripRE, '') : '';
 }
 
 function looseCompareArrays(a, b) {
