diff --git a/node_modules/bootstrap/dist/js/bootstrap.bundle.js b/node_modules/bootstrap/dist/js/bootstrap.bundle.js
index 67ab056..3060cce 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.bundle.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.bundle.js
@@ -5789,9 +5789,10 @@
           this._disposePopper();
         }

-        this._element.removeAttribute('aria-describedby');
-
-        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+        }
       };

       this._queueCallback(complete, this.tip, this._isAnimated());
diff --git a/node_modules/bootstrap/dist/js/bootstrap.esm.js b/node_modules/bootstrap/dist/js/bootstrap.esm.js
index 87a487e..7f5eb42 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.esm.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.esm.js
@@ -3939,9 +3939,10 @@ class Tooltip extends BaseComponent {
         this._disposePopper();
       }

-      this._element.removeAttribute('aria-describedby');
-
-      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+      if (this._element) {
+        this._element.removeAttribute('aria-describedby');
+        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+      }
     };

     this._queueCallback(complete, this.tip, this._isAnimated());
diff --git a/node_modules/bootstrap/js/dist/tooltip.js b/node_modules/bootstrap/js/dist/tooltip.js
index 5e86b8e..2e31ccd 100644
--- a/node_modules/bootstrap/js/dist/tooltip.js
+++ b/node_modules/bootstrap/js/dist/tooltip.js
@@ -291,9 +291,10 @@
           this._disposePopper();
         }

-        this._element.removeAttribute('aria-describedby');
-
-        EventHandler__default.default.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler__default.default.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        }
       };

       this._queueCallback(complete, this.tip, this._isAnimated());
diff --git a/node_modules/bootstrap/js/src/tooltip.js b/node_modules/bootstrap/js/src/tooltip.js
index 748a0e1..4e7c0ba 100644
--- a/node_modules/bootstrap/js/src/tooltip.js
+++ b/node_modules/bootstrap/js/src/tooltip.js
@@ -273,8 +273,10 @@ class Tooltip extends BaseComponent {
         this._disposePopper()
       }

-      this._element.removeAttribute('aria-describedby')
-      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
+      if (this._element) {
+        this._element.removeAttribute('aria-describedby')
+        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
+      }
     }

     this._queueCallback(complete, this.tip, this._isAnimated())
