diff --git a/node_modules/bootstrap/dist/js/bootstrap.bundle.js b/node_modules/bootstrap/dist/js/bootstrap.bundle.js
index 67ab056..66358dc 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.bundle.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.bundle.js
@@ -5683,10 +5683,13 @@
 
     dispose() {
       clearTimeout(this._timeout);
-      EventHandler.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
 
-      if (this._element.getAttribute('data-bs-original-title')) {
-        this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'));
+      if (this._element) {
+        EventHandler.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+
+        if (this._element.getAttribute('data-bs-original-title')) {
+          this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'));
+        }
       }
 
       this._disposePopper();
@@ -5789,9 +5792,10 @@
           this._disposePopper();
         }
 
-        this._element.removeAttribute('aria-describedby');
-
-        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+        }
       };
 
       this._queueCallback(complete, this.tip, this._isAnimated());
@@ -5986,7 +5990,9 @@
         }
       };
 
-      EventHandler.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+      if (this._element) {
+        EventHandler.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+      }
     }
 
     _fixTitle() {
diff --git a/node_modules/bootstrap/dist/js/bootstrap.esm.js b/node_modules/bootstrap/dist/js/bootstrap.esm.js
index 87a487e..4d399e6 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.esm.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.esm.js
@@ -3833,10 +3833,13 @@ class Tooltip extends BaseComponent {
 
   dispose() {
     clearTimeout(this._timeout);
-    EventHandler.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
 
-    if (this._element.getAttribute('data-bs-original-title')) {
-      this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'));
+    if (this._element) {
+      EventHandler.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+
+      if (this._element.getAttribute('data-bs-original-title')) {
+        this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'));
+      }
     }
 
     this._disposePopper();
@@ -3939,9 +3942,10 @@ class Tooltip extends BaseComponent {
         this._disposePopper();
       }
 
-      this._element.removeAttribute('aria-describedby');
-
-      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+      if (this._element) {
+        this._element.removeAttribute('aria-describedby');
+        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+      }
     };
 
     this._queueCallback(complete, this.tip, this._isAnimated());
@@ -4136,7 +4140,9 @@ class Tooltip extends BaseComponent {
       }
     };
 
-    EventHandler.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+    if (this._element) {
+      EventHandler.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+    }
   }
 
   _fixTitle() {
diff --git a/node_modules/bootstrap/dist/js/bootstrap.js b/node_modules/bootstrap/dist/js/bootstrap.js
index 9ccfb0b..63cbc0c 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.js
@@ -3857,10 +3857,13 @@
 
     dispose() {
       clearTimeout(this._timeout);
-      EventHandler.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
 
-      if (this._element.getAttribute('data-bs-original-title')) {
-        this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'));
+      if (this._element) {
+        EventHandler.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+
+        if (this._element.getAttribute('data-bs-original-title')) {
+          this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'));
+        }
       }
 
       this._disposePopper();
@@ -4160,7 +4163,9 @@
         }
       };
 
-      EventHandler.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+      if (this._element) {
+        EventHandler.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+      }
     }
 
     _fixTitle() {
diff --git a/node_modules/bootstrap/js/dist/tooltip.js b/node_modules/bootstrap/js/dist/tooltip.js
index 5e86b8e..b2346a4 100644
--- a/node_modules/bootstrap/js/dist/tooltip.js
+++ b/node_modules/bootstrap/js/dist/tooltip.js
@@ -185,10 +185,12 @@
 
     dispose() {
       clearTimeout(this._timeout);
-      EventHandler__default.default.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+      if (this._element) {
+        EventHandler__default.default.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
 
-      if (this._element.getAttribute('data-bs-original-title')) {
-        this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'));
+        if (this._element.getAttribute('data-bs-original-title')) {
+          this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'));
+        }
       }
 
       this._disposePopper();
@@ -291,9 +293,10 @@
           this._disposePopper();
         }
 
-        this._element.removeAttribute('aria-describedby');
-
-        EventHandler__default.default.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler__default.default.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        }
       };
 
       this._queueCallback(complete, this.tip, this._isAnimated());
@@ -488,7 +491,9 @@
         }
       };
 
-      EventHandler__default.default.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+      if (this._element) {
+        EventHandler__default.default.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler);
+      }
     }
 
     _fixTitle() {
diff --git a/node_modules/bootstrap/js/src/tooltip.js b/node_modules/bootstrap/js/src/tooltip.js
index 748a0e1..cad410c 100644
--- a/node_modules/bootstrap/js/src/tooltip.js
+++ b/node_modules/bootstrap/js/src/tooltip.js
@@ -170,10 +170,12 @@ class Tooltip extends BaseComponent {
   dispose() {
     clearTimeout(this._timeout)
 
-    EventHandler.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler)
+    if (this._element) {
+      EventHandler.off(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler)
 
-    if (this._element.getAttribute('data-bs-original-title')) {
-      this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'))
+      if (this._element.getAttribute('data-bs-original-title')) {
+        this._element.setAttribute('title', this._element.getAttribute('data-bs-original-title'))
+      }
     }
 
     this._disposePopper()
@@ -273,8 +275,10 @@ class Tooltip extends BaseComponent {
         this._disposePopper()
       }
 
-      this._element.removeAttribute('aria-describedby')
-      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
+      if (this._element) {
+        this._element.removeAttribute('aria-describedby')
+        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
+      }
     }
 
     this._queueCallback(complete, this.tip, this._isAnimated())
@@ -480,7 +484,9 @@ class Tooltip extends BaseComponent {
       }
     }
 
-    EventHandler.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler)
+    if (this._element) {
+      EventHandler.on(this._element.closest(SELECTOR_MODAL), EVENT_MODAL_HIDE, this._hideModalHandler)
+    }
   }
 
   _fixTitle() {
