diff --git a/node_modules/bootstrap/dist/js/bootstrap.bundle.js b/node_modules/bootstrap/dist/js/bootstrap.bundle.js
index 111a2ca..af5e87b 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.bundle.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.bundle.js
@@ -5204,8 +5204,10 @@
         if (!this._isHovered) {
           this._disposePopper();
         }
-        this._element.removeAttribute('aria-describedby');
-        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+        }
       };
       this._queueCallback(complete, this.tip, this._isAnimated());
     }
diff --git a/node_modules/bootstrap/dist/js/bootstrap.esm.js b/node_modules/bootstrap/dist/js/bootstrap.esm.js
index 8143ac2..d3bb40c 100644
--- a/node_modules/bootstrap/dist/js/bootstrap.esm.js
+++ b/node_modules/bootstrap/dist/js/bootstrap.esm.js
@@ -3361,8 +3361,10 @@ class Tooltip extends BaseComponent {
       if (!this._isHovered) {
         this._disposePopper();
       }
-      this._element.removeAttribute('aria-describedby');
-      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+      if (this._element) {
+        this._element.removeAttribute('aria-describedby');
+        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN$2));
+      }
     };
     this._queueCallback(complete, this.tip, this._isAnimated());
   }
diff --git a/node_modules/bootstrap/js/dist/tooltip.js b/node_modules/bootstrap/js/dist/tooltip.js
index a531c2a..30ac16d 100644
--- a/node_modules/bootstrap/js/dist/tooltip.js
+++ b/node_modules/bootstrap/js/dist/tooltip.js
@@ -253,8 +253,10 @@
         if (!this._isHovered) {
           this._disposePopper();
         }
-        this._element.removeAttribute('aria-describedby');
-        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        if (this._element) {
+          this._element.removeAttribute('aria-describedby');
+          EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN));
+        }
       };
       this._queueCallback(complete, this.tip, this._isAnimated());
     }
diff --git a/node_modules/bootstrap/js/src/tooltip.js b/node_modules/bootstrap/js/src/tooltip.js
index 1252811..76c4acb 100644
--- a/node_modules/bootstrap/js/src/tooltip.js
+++ b/node_modules/bootstrap/js/src/tooltip.js
@@ -273,8 +273,10 @@ class Tooltip extends BaseComponent {
         this._disposePopper()
       }
 
-      this._element.removeAttribute('aria-describedby')
-      EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
+      if (this._element) {
+        this._element.removeAttribute('aria-describedby')
+        EventHandler.trigger(this._element, this.constructor.eventName(EVENT_HIDDEN))
+      }
     }
 
     this._queueCallback(complete, this.tip, this._isAnimated())
