diff --git a/node_modules/nitropack/dist/runtime/entries/azure-functions.mjs b/node_modules/nitropack/dist/runtime/entries/azure-functions.mjs
index 7437e5b..ce44066 100644
--- a/node_modules/nitropack/dist/runtime/entries/azure-functions.mjs
+++ b/node_modules/nitropack/dist/runtime/entries/azure-functions.mjs
@@ -16,6 +16,6 @@ export async function handle(context, req) {
     // cookies https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-node?tabs=typescript%2Cwindows%2Cazure-cli&pivots=nodejs-model-v4#http-response
     cookies: getAzureParsedCookiesFromHeaders(headers),
     headers: normalizeLambdaOutgoingHeaders(headers, true),
-    body: body ? body.toString() : statusText
+    body: body ? body : statusTex
   };
 }
