diff --git a/node_modules/nuxt/dist/app/composables/manifest.js b/node_modules/nuxt/dist/app/composables/manifest.js
index a41895e..985682e 100644
--- a/node_modules/nuxt/dist/app/composables/manifest.js
+++ b/node_modules/nuxt/dist/app/composables/manifest.js
@@ -10,7 +10,10 @@ function fetchManifest() {
     throw new Error("[nuxt] app manifest should be enabled with `experimental.appManifest`");
   }
   const buildId = useAppConfig().nuxt?.buildId;
-  manifest = $fetch(buildAssetsURL(`builds/meta/${buildId}.json`));
+  manifest = $fetch(buildAssetsURL(`builds/meta/${buildId}.json`), {
+    retry: 30,
+    retryDelay: 1000,
+  });
   manifest.then((m) => {
     matcher = createMatcherFromExport(m.matcher);
   });
