diff --git a/node_modules/vue-google-charts/dist/index.cjs b/node_modules/vue-google-charts/dist/index.cjs
index 625f83b..c9a814b 100644
--- a/node_modules/vue-google-charts/dist/index.cjs
+++ b/node_modules/vue-google-charts/dist/index.cjs
@@ -9,7 +9,8 @@ let chartsLoaderPromise = null;
 const loadedPackages = new Map();
 function getChartsLoader() {
     // If already included in the page:
-    if (window.google !== undefined) {
+    if (window.google !== undefined && window.google.charts) {
+        console.log('Returning 1')
         return Promise.resolve(window.google.charts);
     }
     if (chartsLoaderPromise === null) {
diff --git a/node_modules/vue-google-charts/dist/index.js b/node_modules/vue-google-charts/dist/index.js
index c8dc85d..bc148cf 100644
--- a/node_modules/vue-google-charts/dist/index.js
+++ b/node_modules/vue-google-charts/dist/index.js
@@ -5,7 +5,7 @@ let chartsLoaderPromise = null;
 const loadedPackages = new Map();
 function getChartsLoader() {
     // If already included in the page:
-    if (window.google !== undefined) {
+    if (window.google !== undefined && window.google.charts) {
         return Promise.resolve(window.google.charts);
     }
     if (chartsLoaderPromise === null) {
diff --git a/node_modules/vue-google-charts/legacy/index.cjs b/node_modules/vue-google-charts/legacy/index.cjs
index d359648..a1bc926 100644
--- a/node_modules/vue-google-charts/legacy/index.cjs
+++ b/node_modules/vue-google-charts/legacy/index.cjs
@@ -7,7 +7,7 @@ let chartsLoaderPromise = null;
 const loadedPackages = new Map();
 function getChartsLoader() {
     // If already included in the page:
-    if (window.google !== undefined) {
+    if (window.google !== undefined && window.google.charts) {
         return Promise.resolve(window.google.charts);
     }
     if (chartsLoaderPromise === null) {
diff --git a/node_modules/vue-google-charts/legacy/index.js b/node_modules/vue-google-charts/legacy/index.js
index abe0908..9771527 100644
--- a/node_modules/vue-google-charts/legacy/index.js
+++ b/node_modules/vue-google-charts/legacy/index.js
@@ -3,7 +3,7 @@ let chartsLoaderPromise = null;
 const loadedPackages = new Map();
 function getChartsLoader() {
     // If already included in the page:
-    if (window.google !== undefined) {
+    if (window.google !== undefined && window.google.charts) {
         return Promise.resolve(window.google.charts);
     }
     if (chartsLoaderPromise === null) {
