diff --git a/node_modules/vue-bundle-renderer/dist/runtime.mjs b/node_modules/vue-bundle-renderer/dist/runtime.mjs
index e4bd416..b2f6fae 100644
--- a/node_modules/vue-bundle-renderer/dist/runtime.mjs
+++ b/node_modules/vue-bundle-renderer/dist/runtime.mjs
@@ -4,7 +4,7 @@ const defaultShouldPrefetch = (resource) => resource.resourceType !== "font";
 const defaultShouldPreload = (resource) => ["module", "script", "style"].includes(resource.resourceType || "");
 function createRendererContext({ manifest, buildAssetsURL, shouldPrefetch, shouldPreload }) {
   const ctx = {
-    shouldPrefetch: shouldPrefetch || defaultShouldPrefetch,
+    shouldPrefetch: () => false, // shouldPrefetch || defaultShouldPrefetch,
     shouldPreload: shouldPreload || defaultShouldPreload,
     buildAssetsURL: buildAssetsURL || withLeadingSlash,
     manifest: void 0,
